docs/libraries/azure-resource-manager/rules/delete-operation-response-codes.md

```diff
@@ -0,0 +1,62 @@
+---
+title: delete-operation-response-codes
+---
+
+```text title=- Full name-
+@azure-tools/typespec-azure-resource-manager/delete-operation-response-codes
+```
+
+## Synchronous
+
+Synchronous delete operations must have 200, 204, and default responses. They must not have any other responses.
+
+#### ❌ Incorrect
+
+```tsp
+@armResourceOperations
+interface Employees {
+  @armResourceDelete(Employee)
+  delete(...ApiVersionParameter): {
+    @statusCode _: 200;
+    result: boolean;
+  };
+}
+```
+
+#### ✅ Correct
+
+```tsp
+@armResourceOperations
+interface Employees {
+  @armResourceDelete(Employee)
+  delete(...ApiVersionParameter): {
+    @statusCode _: 200;
```

Comment:
Looks like this wasn't changed, this should use ArmResourceDeleteSync instead of a custom operation.

Reply:
It looks like rebasing threw away my changes. Fun times.